LondonBritishLibrary/orient/BLorient11763.xml

```diff
@@ -0,0 +1,125 @@
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Mediocre, square handwriting.</desc>
+                                <date notBefore="1600" notAfter="1699" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc>One coloured magical picture.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="true">
+                                <decoNote xml:id="b1" type="SlipCase"><material key="leather"/>Contained in a cylindrical leather case.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1699">17th century</origDate>
+                        </origin>
+                        <provenance>Two owners are named: <persName role="owner" ref="PRS14519WaldaSamuel">Walda Sāmuʾel</persName>
+                            and <persName role="owner" ref="PRS14520BakweraSeyon">Bakʷǝra Ṣǝyon</persName></provenance>
+                        <acquisition>Presented by Miss <persName role="bequeather" ref="PRS14518Gresley">M. Gresley</persName>.
+                            <date when="1944-03-28">28 March 1944</date>.</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                            <citedRange unit="page">113</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
```

Comment:
the `keywords `are missing completely, when you edit in a new branch please add `Magic`